src/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -49,11 +49,7 @@
     <Compile Include="System\Text\Json\Reader\Utf8JsonReader.TryGet.cs" />
     <Compile Include="System\Text\Json\Serialization\ClassType.cs" />
     <Compile Include="System\Text\Json\Serialization\ConverterList.cs" />
-    <Compile Include="System\Text\Json\Serialization\Converters\DefaultDerivedDictionaryConverter.cs" />
-    <Compile Include="System\Text\Json\Serialization\Converters\DefaultDerivedEnumerableConverter.cs" />
     <Compile Include="System\Text\Json\Serialization\Converters\DefaultArrayConverter.cs" />
```

Comment:
Are we still going to convert arrays (i.e: `int[]` or `MyClass[]`)?

Reply:
Yes, we use converters for arrays because we don't know how many items the array will have.